### DISCUSSION TITLE
why  fan_in and fan_out return all zeros for all the methods？

### DISCUSSION THREAD

**Discussion by cp3-three** [ishepard/pydriller#247] [Q&A] [answered]
I am getting fan_in and fan_out all zeros for all the methods

**Comment by ishepard** [comment, marked as answer]
Pydriller uses Lizard to calculate metrics, and back in 2019 I opened an https://github.com/terryyin/lizard/issues/250 on Lizard for this. It still hasn't been solved unfortunately.
